### ISSUE TITLE
Word hints are always included.

### ISSUE THREAD

**Issue by blackeangel** [klausw/hackerskeyboard#710]
Please add this function to the settings, or tell me where to edit what in the code - I'll do it myself.

**Comment by TPS** [comment]
@blackeangel The setting already exists ~¾ down main HK Settings page @ "Word suggestion settings" » "Show suggestions" — did you look⁉

**Comment by blackeangel** [comment]
@TPS , Of course, both ticks are. However, in terminal emulators, in search engines, in filling out forms, tooltips are not available.
